lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -193,16 +190,18 @@ class CategoryRenderer {
    */
   renderAuditGroup(group) {
     const groupEl = this.dom.createElement('div', 'lh-audit-group');
-    const summaryEl = this.dom.createChildOf(groupEl, 'div');
-    const summaryInnerEl = this.dom.createChildOf(summaryEl, 'div', 'lh-audit-group__summary');
-    const headerEl = this.dom.createChildOf(summaryInnerEl, 'div', 'lh-audit-group__header');
 
+    // Group header DOM roughly matches the clump header DOM, and that's OK.
```

Comment:
should remove this comment. See reasoning in earlier thread, but, more importantly, this wouldn't be the place to put the comment anyways :)

This group header is using `group` class names, nothing weird about that or worth commenting on; it's the *clump* header using `group` class names that's the aberration.

Reply:
done